### PR TITLE
XadesDateTime

### PR DESCRIPTION
**Pull request by alexey-pelykh**

### CODE REVIEW

File: src/xml/date_time.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    })
+    public Value: Date;
+
+    /**
+     * Format of the datetime value
+     *
+     * Check [dateformat]{@link https://www.npmjs.com/package/dateformat} for supported formatting options.
+     * If not specified, [Date#toISOString()]{@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString} is used.
+     */
+    public Format: string;
```
Comment (microshine) [inline src/xml/date_time.ts]: it  looks `Format` is optional
```ts
public Format?: string;
```

**Comment by microshine** [comment]
@alexey-pelykh Could you update `index.d.ts`? It can be a reason of bad circleci test

**Comment by alexey-pelykh** [comment]
Well, it still fails, unfortunately

**Comment by microshine** [comment]
this is circleci setting. I can't to send coverall result to server. But mocha tests are ok

**Comment by microshine** [comment]
v2.0.10 is available

**Comment by microshine** [comment]
@alexey-pelykh Thank you for you PR

**Pull request merged**